README.md

```diff
@@ -2063,6 +2063,14 @@ Minimum Requirements:
   * Cisco Nexus 31xx, OS Version 7.0(3)I2(1), Environments: Bash-shell, Guestshell
   * Cisco Nexus 30xx, OS Version 7.0(3)I2(1), Environments: Bash-shell, Guestshell
 
+## Cisco IOS Differences
```

Comment:
Nitpick - change this from 'IOS' to 'OS' (and update the hyperlinks accordingly) - NX-OS is an operating system, but it's not IOS.